microprot/scripts/process_fasta.py

```diff
@@ -123,11 +124,53 @@ def read_representatives(represent):
     return sorted(list(ids))
 
 
+def split_fasta(seqs, prefix=None, outdir=None):
```

Comment:
I agree with @sjanssen2 that we'd better use skbio but, I think we already did it. Am I missing something?